lib/pages/new_profile/ok_button_widget.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'new_profile_controller.dart';
+
+class OkButtonWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return FlatButton(
+        child: const Text('ok'),
+        onPressed: () {
+          return context.read<NewProfileController>().postUser();
+        });
```

Comment:
```suggestion
        onPressed: () async {
          await context.read<NewProfileController>().postUser();
        });
```
にしたいすね
returnもいらなさそう

Reply:
thus!!